cloudservice/datadog/datadog_test.go

```diff
@@ -0,0 +1,123 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2022 Datadog, Inc.
+
+package datadog
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+func TestDatadog(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "CloudService Datadog Suite")
+}
+
+var _ = BeforeSuite(func() {
+})
+
+var _ = AfterSuite(func() {
```

Comment:
Same :point_up:

Reply:
It's totally not thanks for noticing 😅 I thought it was some convention but it's completely useless